lib/tasks/fix_past_status_data.rake

```diff
@@ -0,0 +1,117 @@
+# Copyright © 2011-2018 MUSC Foundation for Research Development~
```

Comment:
One more thing I noticed. Can you update the copyright for this file to 2019

Reply:
Yup, done